dev-ruby/hashicorp-checkpoint/hashicorp-checkpoint-0.1.5-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_RECIPE_DOC="none"
```

Comment:
Why did you change this to "none"? It's "rdoc" in the current ebuild, but that it the default in EAPI 7 so it could just be removed altogether.

Reply:
As it is default I removed it to keep it simple